packages/lit-dev-content/site/docs/components/decorators.md

```diff
@@ -102,14 +102,19 @@ assumptions = {
 };
 
 plugins = [
```

Comment:
Not exactly relevant to the PR but since we're already here, I think there should be a colon rather than an equal sign here and for the `assumptions` line above.

Reply:
Hmm... Yeah, this is a good point. These things were copied from different sources and each of them uses a different format. I think to your point below, standardizing on something that would work in either JS or JSON is a solid choice....